tests/controller_tests/test_ingest.py

```diff
@@ -515,12 +515,32 @@ def test_extract_data_extracts_from_multiple_sheets(example_data_model_file):
     assert isinstance(list(workbook.values())[0], pd.DataFrame)
 
 
-def test_next_submission_id_first_submission(test_client):
-    sub_id = next_submission_id(reporting_round=1)
-    assert sub_id == "S-R01-1"
-
-
-def test_next_submission_id_existing_submissions(test_client):
+@pytest.mark.parametrize(
```

Comment:
This line can be tidied to be 
`@pytest.mark.parametrize("reporting_round", [1, 2, 3, 4])`